src/Runner.php

```diff
@@ -57,7 +68,7 @@ public function __construct($roboClass = null, $roboFile = null)
         $this->dir = getcwd();
     }
 
-    protected function errorCondtion($msg, $errorType)
+    protected function errorCondition($msg, $errorType)
```

Comment:
Typo's in the public API are a bummer, as fixing them is technically a b/c break. I think it's going to be rare enough for folks to want to override / call this that we can go ahead and fix it. Could you add a note in the CHANGELOG for this, though, so we don't forget?